Backend/nix/scripts.nix

```diff
@@ -44,7 +44,20 @@ _:
         exec = ''
           set -x
           cd ./Backend  # These processes expect $PWD to be backend, for reading dhall configs
-          cabal run alchemist-generator-exe -- "$@"
```

Comment:
@ErasedSoul I've tested this command on my Linux machine and it did not worked. Did it work for you? 
I suggest to add new command `, run-generator-all`. Please check my changes